wsd/AdminModel.cpp

```diff
@@ -1159,6 +1159,25 @@ void AdminModel::getMetrics(std::ostringstream &oss)
     oss << "error_unauthorized_request " << UnauthorizedRequestException::count << "\n";
     oss << "error_service_unavailable " << ServiceUnavailableException::count << "\n";
     oss << "error_parse_error " << ParseError::count << "\n";
+    oss << std::endl;
+
+    int tick_per_sec = sysconf(_SC_CLK_TCK);
+    // dump document data
+    for (const auto& it : _documents)
+    {
+        const Document &doc = *it.second;
+        std::string suffix = "{pid=" + std::to_string(doc.getPid()) + "} ";
```

Comment:
it is possible to get also the WOPSrc ?

Reply:
i found this code, maybe all of this attributes are from interesst for OpenMetric Labels

```
                << "\"docKey\"" << ':' << '"' << it.second->getDocKey() << '"' << ','
                << "\"fileName\"" << ':' << '"' << encodedFilename << '"' << ','
                << "\"wopiHost\"" << ':' << '"' << it.second -> getHostName() << '"' << ','
```

Reply:
the 'docKey' is the WOPISrc for a wopi-like server. I added the other bits there.